common/app/views/fragments/contentMeta.scala.html

```diff
@@ -65,17 +65,17 @@
         }
     }
 }
-
-<div class="content__meta-container js-content-meta js-football-meta u-cf
-    @if(item.trail.byline.isEmpty){ content__meta-container--no-byline}
-    @if(item.tags.isLiveBlog) { content__meta-container--liveblog}
-    @if(item.elements.hasShowcaseMainElement){ content__meta-container--showcase}
-    @if(item.content.hasTonalHeaderByline){ content__meta-container--tonal-header}
-    @item.content.contributorBio.map { bio => content__meta-container--bio}
-    @if(item.content.isExplore) {content__meta-container--explore}
-    @if(item.tags.contributors.length == 1 && item.tags.contributors.headOption.exists(_.properties.twitterHandle.nonEmpty)) { content__meta-container--twitter}
-    @if(item.tags.contributors.length == 1 && item.tags.contributors.headOption.exists(_.properties.emailAddress.nonEmpty)) { content__meta-container--email}">
-
+<div class="@RenderClasses(Map(
+    ("content__meta-container--no-byline", item.trail.byline.isEmpty),
+    ("content__meta-container--liveblog", item.tags.isLiveBlog),
+    ("content__meta-container--float", item.elements.hasShowcaseMainElement || isPaidContent(page)),
```

Comment:
cc @guardian/commercial-dev not sure if this is the right way to check this sort of page: https://www.theguardian.com/visit-las-vegas/2017/apr/12/from-absinthe-to-zumanity-the-20-best-shows-in-las-vegas